File: ReadingTracker/app/src/main/java/com/codelab/readingtracker/todo/TodoItemView.java
package com.codelab.readingtracker.todo;

import android.content.Context;
import android.view.View;
import android.widget.TextView;

import com.codelab.readingtracker.R;

public class TodoItemView extends View {

    TextView taskTextView;

    public TodoItemView(Context context) {
        super(context);
        inflate(context, R.layout.view_todo_item, null);
        taskTextView = findViewById(R.id.todo_item_textView);
    }

    public void setTask(String task) {
        if(taskTextView != null) {
            taskTextView.setText(task);
        }
    }
}

File: ReadingTracker/app/src/main/java/com/codelab/readingtracker/todo/TodoItemView.kt
package com.codelab.readingtracker.todo

import android.content.Context
import android.util.AttributeSet
import android.widget.LinearLayout
import com.codelab.readingtracker.R
import kotlinx.android.synthetic.main.view_todo_item.view.*

class TodoItemView @JvmOverloads constructor(
        context: Context, attrs: AttributeSet? = null, defStyleAttrs: Int = 0
) : LinearLayout(context, attrs, defStyleAttrs) {

    init {
        inflate(context, R.layout.view_todo_item, this)
    }

    fun setTask(task: String) {
        if (todo_item_textView != null) {
            todo_item_textView!!.text = task
        }
    }
}
